go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/cyberark/conjur-api-go v0.5.2
 	github.com/cyberark/conjur-authn-k8s-client v0.13.0
 	github.com/cyberark/summon v0.7.0
+	github.com/denisenkom/go-mssqldb v0.0.0-20191001013358-cfbb681360f0
```

Comment:
the changes to go.* are unrelated to this code, afaict. do they need to be included in this PR?